scalameter-core/src/main/scala/org/scalameter/MeasureBuilder.scala

```diff
@@ -38,12 +43,12 @@ class MeasureBuilder[T, U](
   
       if (ctx(Key.exec.requireGC)) compat.Platform.collectGarbage()
   
-      val measurements = measurer.measure[T, U](
+      val measurements = measurer.measure[T](
         ctx, ctx(Key.exec.benchRuns),
         setup, teardown, regen, b
       )
   
-      resultFunction(measurements)
+      resultFunction(measurements.map(_.value))
```

Comment:
What you suggest could be a way. How would you lift `Seq[T] => T` to `Seq[Quantity[T]] => Quantity[T]` without knowing the unit, though?

I still don't get what is causing the error... And you say you deleted both `target` directories in the root, and the `scalameter-core`?